recruiter/tests.py

```diff
@@ -1,3 +1,46 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from django.db.models import QuerySet
 
-# Create your tests here.
+from recruiter.models import Company, Recruiter
+
+
+@pytest.fixture
+def create_company():
```

Comment:
see comments I placed on other PRs - fixture names souhld be nouns not verbs.

Reply:
Will change the names now